src/map/atcommand.c

```diff
@@ -7076,7 +7076,7 @@ ACMD_FUNC(mobinfo)
 #ifdef VIP_ENABLE
 			// Display item rate increase for VIP.
 			if (pc_isvip(sd) && battle_config.vip_drop_increase)
-				droprate += battle_config.vip_drop_increase;
+				droprate += (droprate * battle_config.vip_drop_increase) / 100;
```

Comment:
Before, we were doing +vip_drop_increase constant to every drop in @mobinfo, which was incorrect. Fixed this to add vip_drop_increase as % of current droprate adjusted by server rate.
vip_drop_increase = 50 before was showing in @mobinfo poring as:
Apple 60.5% (instead of 60%), Unripe Apple: 0.7% (instead of 0.2%), Poring Card 0.51% (instead of 0.01%)